ios/TestApp/TestApp/main.m

```diff
@@ -0,0 +1,8 @@
+#import <UIKit/UIKit.h>
```

Comment:
nit: UIKit import is redundant, since it is already imported by AppDelegate anyway